src/vellum/workflows/nodes/core/map_node/tests/test_node.py

```diff
@@ -56,7 +56,6 @@ class Outputs(BaseOutputs):
             value: int
 
         def run(self) -> Outputs:
-            time.sleep(0.03)
```

Comment:
Revert all changes in this PR. The only changes should be in the `examples/workflows` directory